dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3944,7 +3944,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_WORKER_REGISTER_LEASE_ENABLED =
       booleanBuilder(Name.MASTER_WORKER_REGISTER_LEASE_ENABLED)
-          .setDefaultValue(true)
+          .setDefaultValue(false)
```

Comment:
it's fine to disable it now, i think this leasing logic needs a bit of rework in dora

Reply:
well worker registration in 3.0 isn't as heavy-lifting as it used to be in 2.x so probably we have a chance to get rid of it completely